### PR TITLE
IRO-1699 - Home page

### PR DESCRIPTION
**Pull request by ndemidovich-sc**

### CODE REVIEW

**Comment by brekk** [comment]
CleanShot 2022-05-04 at 10 34 45@2x
Weird image overflow at 769px wide.

File: components/NavSearch/DemoSearchComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+          ) : null}
+          <Input
+            {...InputProps}
+            value={val ? val.label : search}
+            ref={inputRef as RefObject<HTMLInputElement>}
+            onFocus={(e) => e.target?.select()}
+            onChange={(e) => {
+              if (val) {
+                setVal(null);
+              }
```
[the remark this answers is not in the document]
Comment (azhurauski-sc) [reply components/NavSearch/DemoSearchComponent.tsx:180]: Its used to clear selected value when start to input new one.

**Comment by azhurauski-sc** [comment]
Fixes from dev notes:

- Logo in header move up for 2px
- Header border left only on bottom
- Hamburger icon improved
- Alignment improved

Other fixes  will be done with `ui-kit` library update

File: components/Layout/NavMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          bgColor: $bg,
+          transform: showMenu ? "rotate(45deg)" : "translateY(0.2575rem)",
+        }}
+        _after={{
+          content: `""`,
+          position: "absolute",
+          ...burgerLineStyle,
+          bgColor: $bg,
+          transform: showMenu ? "rotate(-45deg)" : "translateY(-0.2575rem)",
+        }}
```
[the remark this answers is not in the document]
Comment (azhurauski-sc) [reply components/Layout/NavMenu.tsx]: moved common part to constant except `transform`

**Comment by brekk** [comment]
#### Callouts
- Fonts are currently incorrect, it seems like it comes from within `ui-kit` -- I made a separate branch but wasn't able to get it working without hacking it.

#### Nav Menu
- Much better than before, click areas are great now

#### BlockInfoCards
- Right now these break into two columns at 480px / the first breakpoint
  -> Can this change be pushed to something like 576px / 36rem?
  -> Right now it's too tight / too early to break into two columns
- Can we have some more spacing around the images in the cards? Right now the text is sometimes much too tight.

#### LatestBlocks
- The padding on the labeled values is too wide IMO, something like 1.5rem instead of 2rem padding makes things look a lot 
- At whatever breakpoint 1079px is nearest, right now the date and the time are sometimes on the same line and sometimes they are two lines -- I think splitting them into two separate fields and having them sit atop one another (with an explicit line break in between) would be good

#### Search bar
- Not sure if this is complete (guessing it is not), but it needs a bit of love still (auto-hide on click away / needs keyboard nav eventually)

However, since I think it would likely be easier to get this branch out and then iterate on it further (especially as I have more bandwidth for this project soon) -- I'm inclined to say we should merge this as is and then continue to address feedback.

**Review by brekk — APPROVED** [review]

**Comment by azhurauski-sc** [comment]
Created [IRO-1922](https://linear.app/ironfish/issue/IRO-1922/block-explorer-improvements) with subtasks for commented problems.

**Pull request merged**